### ISSUE TITLE
Reordering

### ISSUE THREAD

**Issue by shaunc** [mharris717/ember-drag-drop#5]
Thanks for these components!

It would be nice to be able to use this for reordering elements of a single list. However, as it is, no information about where the object is dropped is passed via the action. Looking under the hood a bit, in draggable-object-target/handleDrop, the `event.originalEvent.toElement` seems to have the DOM node where something was dropped. A higher level interface would be nice -- where an indication about on or between which content another element was dropped was passed via the action.

**Comment by knownasilya** [comment]
Would also love to see this feature imlemented, like in this [jsbin](http://emberjs.jsbin.com/kenum/3/edit?html,js,output).

**Comment by henryw14** [comment]
I needed something kind of similar. When I dropped an item into a new drop-target, I wanted to remove it from it's previous target. I managed to do this by:
* Extending draggable-object and having it take its current draggable-object target as a handlebars attribute with a `moved(newTarget)` function:
```
import DraggableObject from 'smallbiz/components/draggable-object';

export default DraggableObject.extend({
    tagName: "div",
    classNames: 'external-event navy-bg',
    moved: function(destination){
       ...domain logic for removing itself from the old target if 
        the destination is different and then updating its target
        to the new destination
    }
});
```
* extending draggable-object-target and overriding its handlePayload(payload) function:
```
handlePayload: function(payload){
        var obj = this.get('coordinator.objectMap').getObj(payload);
        obj.ops.source.moved(this);
        this._super(payload);
}
```
If you think a solution like this would be universally useful, I wouldn't mind making a pull request where moved is an empty function that can be overridden

**Comment by umurgdk** [comment]
I handle reordering by wrapping draggable object with drop target. It solved my problem but I can't show placeholders. I tried to show it with css ::before. But somehow it enters a weird loop of add/remove **accepts-drag** class :(

**Comment by umurgdk** [comment]
Ok totally solved the problem. I think there is no need a change on the codebase. I put drop-targets between each item. And I put placeholder div to show some message like "Move here" in each drop-target. When drag over them on the css side Im making placeholder visible. The key point to avoid loop to add 'pointer-events: none' to placeholder div that inside of these drop-targets.

**Comment by dgavey** [comment]
I have implemented a new sortable object on v0.2.0 check it out, hopefully will work for you guys.  I'll be updating with some new features soon.  I'm certainly open to suggestions on how to improve it.
